src/api/dirmonitor/inotify.c

```diff
@@ -37,8 +37,8 @@ int get_changes_dirmonitor(struct dirmonitor_internal* monitor, char* buffer, in
 
 
 int translate_changes_dirmonitor(struct dirmonitor_internal* monitor, char* buffer, int length, int (*change_callback)(int, const char*, void*), void* data) {
-  for (struct inotify_event* info = (struct inotify_event*)buffer; (char*)info < buffer + length; info = (struct inotify_event*)((char*)info + sizeof(struct inotify_event)))
```

Comment:
might be worth documenting this as a comment

Reply:
You mean keeping the loop commented, or adding a comment that documents why we're not looping?

Reply:
don't need to leave the loop in, just explain why only the first value of the buffer is read